src/components/buttonv2/ButtonV2.tsx

```diff
@@ -0,0 +1,232 @@
+import * as React from 'react'
+import * as menu from '@zag-js/menu'
+import clsx from 'clsx'
+import chevronDown from '../assets/chevron-down-16.svg'
+import classes from './styles.module.css'
+import { useMachine, normalizeProps, Portal } from '@zag-js/react'
```

Comment:
can you check your prettier config on why there is a gap between the brackets { useMachine }

Reply:
also change the folder name to button-v2, let's keep it consistent across the repo